app/views/issues/index.html.haml

```diff
@@ -1,20 +0,0 @@
-.scroll-panel-header
-  = link_to "Nowy numer", new_issue_path, class: %w{btn btn-default btn-sm pull-right}
```

Comment:
Niesamowicie dużo błędów się pojawiło i konfliktów. Nie wiem skąd się one
znalazły, nie wiem jak to naprawić. Chciałam zrobić git pull, ale musiałam
najpierw zrobić commit i pojawiły się te błędy.

2016-01-28 19:15 GMT+01:00 Aleksander Smywiński-Pohl <
notifications@github.com>:

> In app/views/issues/index.html.haml
> <https://github.com/apohllo/rocznik/pull/242#discussion_r51162113>:
>
> > @@ -1,20 +0,0 @@
> > -.scroll-panel-header
> > -  = link_to "Nowy numer", new_issue_path, class: %w{btn btn-default btn-sm pull-right}
>
> Proszę przywrócić ten plik.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apohllo/rocznik/pull/242/files#r51162113>.
>